README.md

```diff
@@ -541,6 +541,18 @@ config :wallaby,
   ]
 ```
 
+### Custom Chrome binary
+
+By default chromedriver will find chrome for you but if you want to test against a different version
+you may use this option to point to the other chrome binary.
+
+```elixir
+config :wallaby,
+  chrome: [
```

Comment:
What would you think if we used the driver name as the key? Then it would look something like this.

```elixir
config :wallaby, Wallaby.Experimental.Chrome,
  binary: "path/to/google/chrome"
```
I'm thinking it could be a good convention to use the module name for driver specific config going forward. @keathley, @PragTob Thoughts?

Reply:
:+1: very good idea aaron! :)